5.0/en/0x05-For-Users-Of-4.0.md

```diff
@@ -1,86 +1,84 @@
+### Reduced Coupling with NIST Digital Identity Guidelines
 
-The NIST's [Digital Identity Guidelines (SP 800-63)](htps://pages.nist.gov/800-63-3/) has been and continues to be an excellent, evidence-driven standard for key controls around authentication and authorization. Certain chapters in version 4.x were very closely coupled with these guidelines, including structure and terminology.
+The NIST [Digital Identity Guidelines (SP 800-63)](https://pages.nist.gov/800-63-3/) have long served as a reference for authentication and authorization controls. In version 4.x, certain chapters were closely aligned with NIST’s structure and terminology.
 
-Whilst these guidelines and their upcoming improvements have continued to be an important reference and the basis for many requirements, the strict coupling caused challenges which led to the decision to move away from this approach. These challenges included terminology which was less widely recognised, duplication of very similar requirements in very slightly different situations, and the fact that the mapping was incomplete based on what was perceived to be relevant for ASVS.
+While these guidelines remain an important reference, strict alignment introduced challenges, including less widely recognized terminology, duplication of similar requirements, and incomplete mappings. Version 5.0 moves away from this approach to improve clarity and relevance.
 
-### Moving away from Common Weakness Enumeration (CWE)
+### Moving Away from Common Weakness Enumeration (CWE)
 
-The [Common Weakness Enumeration (CWE)](https://cwe.mitre.org/) from MITRE is a useful way to map out different security weaknesses in software. There are some difficulties in using it including certain CWEs which are categories only and shouldn't be used for mapping, the difficulties in mapping certain existing requirements to a single CWE, and also the fact that there were some loose or inexact mappings in version 4.x of ASVS.
+The [Common Weakness Enumeration (CWE)](https://cwe.mitre.org/) provides a useful taxonomy of software security weaknesses. However, challenges such as category-only CWEs, difficulties in mapping requirements to a single CWE, and the presence of imprecise mappings in version 4.x have led to the decision to discontinue direct CWE mappings in version 5.0.
 
-## Rethinking the level definitions
+## Rethinking Level Definitions
 
-Version 4.x of the ASVS describes the levels as L1 - "Minimum", L2 - "Standard", and L3 - "Advanced" with the implication that all applications processing sensitive data should be at least L2.
+Version 4.x described the levels as L1 ("Minimum"), L2 ("Standard"), and L3 ("Advanced"), with the implication that all applications handling sensitive data should meet at least L2.
 
-There were various issues with this approach that version 5.0 was specifically designed to address.
+Version 5.0 addresses several issues with this approach:
 
-### Easier entry level
+### Improved Entry Level
```

Comment:
I think "Improved" does not send the message that "Level 1 is optimized to have (close to) only 1st layer of defence requirements to make it easier to start with".

Reply:
It should say Easier I think